src/main/java/frc/robot/subsystems/ElbowSubsystem.java

```diff
@@ -190,4 +190,8 @@ public void nudgeElbowUp() {
 		positionController.requestPosition(getActualPosition() + NUDGE_COUNT);
 	  }
 
+	  public void forgetHome() {
+		  positionController.forgetHome();
+	  }	  
+	  
 }
```

Comment:
Isn't the main need for velocity control to overcome the cable carrier on the elbow?  If so, wouldn't the elbow controller also need to the settings config to include the velocity option?

Reply:
it definitely does. i initially left it out because i was planning to try this just for the arm. but then i added homing too, so they both need that. i later decided to make it all configurable, so i'll add it to elbow too

Reply:
No need to rush if the plan was to test with one then move to the other.